superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -217,6 +242,7 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
       width={confirmChange ? '432px' : ''}
       height={confirmChange ? 'auto' : '480px'}
```

Comment:
This probably answers [my prior question](https://github.com/apache/superset/pull/14844/files#r640379087) about the 300px height :/

Reply:
yep!

> Left some comments of varied importance, but there's one thing I noticed while testing.
> 
> The good news is, we can now see each page of the datasets!
> 
> The bad news is, the table header sorting only sorts the particular page of results you're looking at, not the _whole set_
> 
> ![pagesort](https://user-images.githubusercontent.com/812905/119795706-94243880-be8d-11eb-88de-76a75cc4ea42.gif)
> 
> For an example approach, the CRUD views work nicely, in that if you are sorting alphabetically, the first items on page one are _all_ the `A`s, and the last item on the last page are all the `Z`s.
> 
> I think it's worth solving this in a followup PR, where clicking the sort header changes the orderBy parameter on the query itself.
> 
> Side note: (cc @junlincc here) the sort on `name` doesn't seem to work at all, really. That's not the fault of this PR, as far as I see. Can you confirm, and if that's the case, file an Issue? Thanks!

Yes i'll create a ticket to do a follow up for server sorting to keep the scope of pagination with this pr.